doc/metadata.yaml

```diff
@@ -1,5 +1,5 @@
 site_title: LXD
-site_logo_url: https://linuxcontainers.org/static/img/containers.png
+site_logo_url: https://documentation.ubuntu.com/lxd/en/latest/_static/tag.png
```

Comment:
Does anybody know what this file is used for?
The doc build doesn't need it.

Reply:
I'm sorry I have no idea and I couldn't track down what uses it.

Reply:
What does the git history show for this when it was added?

Reply:
It is what provides the link to the logo shown on https://documentation.ubuntu.com/lxd/en/latest/

Reply:
No, that's in the header: https://github.com/canonical/lxd/blob/dff86455dc9e85de010687aa6224538106da4340/doc/.sphinx/_templates/header.html#L179

I think this file was used to provide the navigation for documentation-builder. Probably we just forgot to remove it.

Reply:
I've open PR #11981 for that.